modules/programs/rofi.nix

```diff
@@ -410,6 +410,15 @@ in {
           Cannot use the rofi options 'theme' and 'colors' simultaneously.
         '';
       }
+      (hm.assertions.assertPlatform "programs.rofi" pkgs platforms.linux)
+      {
+        assertion = (!builtins.hasAttr "override" cfg.package)
+          || (cfg.plugins == [ ])
+          || (cfg.package.override.__functionArgs ? plugins);
```

Comment:
Assertions are triggered when false so this will need to be changed.

https://nixos.org/manual/nixos/stable/index.html#sec-assertions-assertions

Reply:
Correct me if I am wrong, but I think that condition is correct.  
Assertion should be triggered when 3 conditions are met at once
1.  package has `override` attr
2. `plugins` is not empty
3. package does not support plugins